test/cmd/images_tests.sh

```diff
@@ -238,7 +238,8 @@ os::test::junit::declare_suite_start "cmd/images${IMAGES_TESTS_POSTFIX:-}/delete
 # test deleting a tag using oc delete
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .spec.tags 0).name}}'" '5.16'
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .status.tags 0).tag}}'" 'latest'
-os::cmd::expect_success "oc delete istag/perl:5.16 --context='${cluster_admin_context}'"
+project="$( oc project -q )"
```

Comment:
if the cluster_admin_context was pointing to the default namespace, that means the project created for the test hadn't showed up in `list projects` yet when the cluster admin logged in at the top of the file

fix that by pinning the namespace in the `cluster_admin_context="$( oc config current-context )"` step